doc/src/manual/strings.md

```diff
@@ -348,6 +348,36 @@ x
 y
 ```
 
+Strings in Julia can contain invalid UTF-8 code unit sequences. This convention allows to
+treat any byte sequence as a `String`. In such situations a rule is that characters are
+formed by the longest sequence of code units that could be a start of some valid code point.
+This rule is best explained by an example:
+
+```jldoctest unicodestring
+julia> s = "\xc0\xa0\xe2\x88\xe2|"
+"\xc0\xa0\xe2\x88\xe2|"
+
+julia> foreach(display, s)
+'\xc0\xa0': [overlong] ASCII/Unicode U+0020 (category Zs: Separator, space)
+'\xe2\x88': Malformed UTF-8 (category Ma: Malformed, bad data)
+'\xe2': Malformed UTF-8 (category Ma: Malformed, bad data)
+'|': ASCII/Unicode U+007c (category Sm: Symbol, math)
+
+julia> isvalid.(collect(s))
+4-element BitArray{1}:
+ false
+ false
+ false
+  true
+```
+
+We can see that first two code units in `s` form an overlong encoding of space character.
+It is invalid, but is accepted in a string as a single character.
+The next two code units form a valid start of a three-byte UTF-8 sequence. However, the fifth
+code unit `\xe2` is not its valid continuation. Therefore code units 3 and 4 form a second
+malformed character in this string. Similarly code unit 5 forms a malformed character because
```

Comment:
I think changing `Therefore code units 3 and 4 form a second malformed character in this string` to `Therefore code units 3 and 4 are also interpreted as malformed characters in this string` could make this sentence a little cleaner.